postgres/Dockerfile

```
@@ -0,0 +1,35 @@
+ARG PG_MAJOR
+ARG PGVECTOR_TAG=0.8.0
```

```suggestion
ARG PGVECTOR_TAG
```
Tracked in versions.yaml